i18n/es-MX/tasks.ftl

```diff
@@ -1,4 +1,4 @@
-tasks = Tareas
+asks = Tareas
```

Comment:
This key was modified, I believe this was a mistake

Reply:
Oops! Fixed it.